tests/README.md

```diff
@@ -1,12 +1,38 @@
 # Oasis Indexer Tests
 
-This directory contains, primarily, e2e tests for the Oasis Indexer. They have been organized as follows:
+This directory contains tests for the Oasis Indexer. They have been organized as follows:
 
+- `e2e/` contains tests that run against a fake blockchain (provided by `oasis-net-runner`) and a black-box indexer (i.e. only external APIs are accessed).
 - `genesis/` contains tests that validate our database state is correct against `oasis-node` genesis state.
 - `http/` contains tests that validate API endpoints behave as expected and return correct output.
 
-## Setup for E2E Tests
+## E2E Tests
 
-To ensure that tests behave as expected, you should have either the local environment or Docker environment configured as per the [top-level README](../README.md#docker-development).
+To ensure that tests behave as expected, you should have Docker environment configured as per the [top-level README](../README.md#docker-development).
 
-Then you can run these tests from the repository root with `make test-ci` or directly from this directory with `go test ./... -v`.
+### Running e2e tests locally
+
+For the canonical way to run these tests, see the [CI config](../../.github/workflows/ci-test.yaml).
+Here are some useful one-liners derived from those commands, and adapted for running locally and repeatedly.
+
+### Get docker containers with current code up&running
+```sh
+make docker &&
+{ docker compose -f tests/e2e/docker-compose.e2e.yml down -t 0 || true; } &&
```

Comment:
This should probably be a part of `clean-e2e`? With the addition of the `-v` switch you proposed.